msteams-platform/concepts/design/personal-apps.md

```diff
@@ -134,7 +134,7 @@ When you select the back button in a Teams app, you will return to the Teams pla
 
 To navigate within the app, configure the back button so that when you select the back button, you can go back to previous steps and navigate within the app.
 
-To **Configure back button**, call [registerBackButtonHandler](/javascript/api/@microsoft/teams-js/pages.backstack?view=msteams-client-js-latest&preserve-view=true&branch=pr-en-us-6801&preserve-view=true) API, which handles the functionality of the back button depending on one of the following conditions:
+To **Configure back button**, call [registerBackButtonHandler](/javascript/api/@microsoft/teams-js/pages.backstack?view=msteams-client-js-latest&preserve-view=true&preserve-view=true) API, which handles the functionality of the back button depending on one of the following conditions:
```

Comment:
Try to avoid *preserve-view* query strings in links. All you need is: /javascript/api/@microsoft/teams-js/pages.backstack

Reply:
Thanks. Updated in latest commit shortly.